core/escrow/escrow.go

```diff
@@ -276,3 +276,46 @@ func IsPaidin(ctx context.Context, configuration *config.Config, contractID *esc
 	}
 	return signedPayinRes.Status.Paid, nil
 }
+
+func Balance(ctx context.Context, configuration *config.Config) (int64, error) {
```

Comment:
if change the function to `func Balance(ctx context.Context, pubKeyRaw []byte) (int64, error)` , it will be better for function, anyway, the code should work as designated.